.github/workflows/release.yaml

```diff
@@ -68,6 +68,17 @@ jobs:
           - os: macos-latest-xlarge
             target: aarch64-apple-darwin
             code-target: darwin-arm64
+          - os: ubuntu-20.04
+            target: x86_64-unknown-linux-musl
+            code-target: linux-x64
+          - os: ubuntu-20.04
+            target: aarch64-unknown-linux-musl
+            code-target: linux-arm64
+            arch: aarch64
+          - os: ubuntu-20.04
+            target: armv7-unknown-linux-musleabihf
+            code-target: linux-armhf
+            arch: armv7
```

Comment:
I think this isn't quite right because these `code-target` values are the same as the ones above. We need to support `alpine-x64` and `alpine-arm64`. See: https://code.visualstudio.com/api/working-with-extensions/publishing-extension#platformspecific-extensions.

Reply:
I need to play around with things (https://github.com/astral-sh/ruff-vscode/pull/387) to understand how this works. I think we need the extension itself to be built under Alpine.

Reply:
@diceroll123 -- Okay, I think I got this working [here](https://github.com/astral-sh/ruff-vscode/pull/387/files) -- notice how the build is done within an Alpine container, does that make sense?

Reply:
Absolutely! Awesome stuff. 😄 Learning a lot from ya.

Reply:
Thanks for doing this, and sorry that it didn't get merged!